temporal/api/cloud/nexus/v1/message.proto

```diff
@@ -0,0 +1,122 @@
+syntax = "proto3";
+
+package temporal.api.cloud.nexus.v1;
+
+option go_package = "go.temporal.io/api/cloud/nexus/v1;nexus";
+
+import "google/protobuf/timestamp.proto";
+
+message IncomingServiceSpec {
+    // The name of the incoming service. Must be unique within an account.
+    // The name must match `[a-zA-Z_][a-zA-Z0-9_]*`.
+    // This field is mutable.
+    string name = 1;
+
+    // The namespace to route requests to. This field is mutable.
+    string namespace = 2;
+
+    // The task queue to route requests to. This field is mutable.
+    string task_queue = 3;
+
+    // The set of authorization policies for the service. Each request's caller
+    // must match with at least one of the specs to be accepted by the incoming service.
+    repeated IncomingServiceAuthorizationSpec authorization_specs = 4;
+}
+
+message IncomingServiceAuthorizationSpec {
+    // The policy set on this incoming service authorization spec.
+    // Possible values are: allowed_cloud_namespace
+    string policy_type = 1;
+
+    // A policy spec that allows one caller namespace to access the service.
+    AllowedCloudNamespacePolicySpec allowed_cloud_namespace_policy_spec = 2;
+}
+
+message AllowedCloudNamespacePolicySpec {
+    // The namespace that is allowed to call into this service. Calling namespace must be in same account as incoming service.
+    string namespace = 1;
+}
+
+// A binding to a cloud namespace and task queue for dispatching incoming Nexus requests.
+message IncomingService {
+    // The id of the incoming service. This is generated by the server and is immutable.
+    string id = 1;
+
+    // The current version of the incoming service specification.
+    // The next update operation must include this version.
+    string resource_version = 2;
+
+    // The incoming service specification.
+    IncomingServiceSpec spec = 3;
+
+    // The current state of the service.
+    // Possible values: activating, activationfailed, active, updating, updatefailed, deleting, deletefailed, deleted
+    // For any failed state, reach out to Temporal Cloud support for remediation.
+    string state = 4;
+
+    // The id of the async operation that is creating/updating/deleting the service, if any.
+    string async_operation_id = 5;
+
+    // The date and time when the service was created.
+    google.protobuf.Timestamp created_time = 6;
+
+    // The date and time when the service was last modified.
+    // Will not be set if the service has never been modified.
```

Comment:
Curious why not default this to `created_time`? Is this for consistency with other APIs?

Reply:
yeah, this is just for consistency. could have definitely been defaulted to created_time here